all-imports.html

```diff
@@ -0,0 +1,12 @@
+<!--
```

Comment:
for index.html, I'd strip it to include a `<meta http-equiv="refresh" content="0;url=demo/" />`